style.css

```diff
@@ -1,18 +1,160 @@
-body {
-
-background-color: gray;
-color: whitesmoke;
-font-size: 100%;
-border-bottom: 2px solid black;
-font-weight: normal;
-padding-bottom: 5px;
-}
-=======
+ body{
+    margin: 0;
+    font-family: initial, sans-serif;
+    font-size: 18px;
+    line-height: 1.6;
+    color: #fff;
     background-color: gray;
-    color: whitesmoke;
-    font-size: 100%;
-    border-bottom: 2px solid black;
-    font-weight: normal;
-    padding-bottom: 5px;
+ *}
+ *:before,
+ *:after{
+    box-sizing: border-box;
+ }
+
+ h1, h2, h3, h4, h5, h6 {
+    margin: 0;
+ }
+
+ /*Contaiter*/
+ .container{
+    width: 100%;
+    max-width: 1200px;
+    margin: 0 auto;
+ }
+
+ /* Intro*/
+ .intro{
+   justify-content: center;
+   display: flex;
+   align-items: center;
+   height: 40em;
+   
+ }
+
+
+ .intro_title {
+    color: #fff;
```

Comment:
Try to keep code stile everywere in code.

Reply:
ok